index.html

```diff
@@ -2,310 +2,1018 @@
+        </div>
+
+        <i class="fa fa-bars text-light d-lg-none"></i>
+      </div>
+    </div>
+
+    <nav class="d-none d-md-none d-lg-block">
+      <div class="nav-links">
+        <ul class="d-flex align-items-center list-unstyled link-list font-roboto social border-bottom">
+          <li class="border-end">
+            <a href="#" class="text-dark">U.S.</a>
+          </li>
+          <li class="border-end">
+            <a class="text-dark" href="#">World</a>
+          </li>
+          <li class="border-end">
+            <a class="text-dark" href="#">Business</a>
+          </li>
+          <li class="border-end">
+            <a class="text-dark" href="#">Tech & Science</a>
+          </li>
+          <li class="border-end">
+            <a class="text-dark" href="#">Culture</a>
+          </li>
+          <li class="border-end">
+            <a class="text-dark" href="#">Newsgeek</a>
+          </li>
+          <li class="border-end">
+            <a class="text-dark" href="#">Sports</a>
+          </li>
+          <li class="border-end">
+            <a class="text-dark" href="#">Health</a>
+          </li>
+          <li class="border-end">
+            <a class="text-dark" href="#">The Debate</a>
+          </li>
+          <li class="border-end">
+            <a class="text-dark" href="#">Experts</a>
+          </li>
+          <li class="border-end">
+            <a class="text-dark" href="#">Vantage</a>
+          </li>
+          <li>
+            <div>
+              <form action="./">
+                <input id="search" type="text" class="search font-roboto" placeholder="Search">
+                <label for="search"><i class="fa fa-search"></i></label>
+              </form>
+            </div>
+          </li>
+        </ul>
+
+        <ul
+          class="c list-group list-group-horizontal list-unstyled bg-white border-bottom link-list social font-roboto">
+          <li><a href="#" class="text-dark">TRENDING</a></li>
+          <li><a href="#" class="text-secondary">JOE BIDEN</a></li>
+          <li><a href="#" class="text-secondary">DONALD TRUMP</a></li>
+          <li><a href="#" class="text-secondary">INAUGURATION</a></li>
+          <li><a href="#" class="text-secondary">CORONAVIRUS</a></li>
+          <li><a href="#" class="text-secondary">STIMULUS PACKAGE</a></li>
+          <li><a href="#" class="text-secondary">VACCINE</a></li>
+          <li><a href="#" class="text-secondary">CAPITOL RIOTS</a></li>
+        </ul>
+
+        <div class="y">
+          <div class="d-flex justify-content-end inline">
+            <div class="triangle"></div>
+            <div class="rectangle"></div>
+          </div>
+        </div>
+      </div>
+    </nav>
+  </header>
+
+  <main>
+    <div class="row">
+      <aside class="col-12 col-md-4 col-lg-3 order-2 order-md-1">
+        <section class="featured-stories pt-3">
+          <header>
+            <span class="header-title">FEATURED STORIES</span>
+          </header>
```

Comment:
Please remove the header from the main section.

Reply:
resolved :heavy_check_mark: